docs/notes/notes.rst

```diff
@@ -5,62 +5,40 @@
 Information about the release lifecycle can be found
 `here <https://github.com/eProsima/Fast-DDS/blob/master/RELEASE_SUPPORT.md>`_.
 
-Version 2.13.5
-==============
-
-This release includes the following **features** in an ABI compatible manner:
-
-#. New :ref:`property_max_message_size` property to limit output datagrams size
+Version 2.13.6 (EOL)
+====================
 
 This release includes the following **improvements**:
 
-#. Improve ThreadSettingsQoS logging
-#. Allow processing of AckNack submessages with ``count == 0``
-#. Internal refactor on port handling
-#. Do not require ``PYTHON_VERSION`` to be defined in .bat files
-#. Use ``%*`` instead of loop in ``.bat`` scripts
-#. Consider library behavior changes as ABI breaks in the PR template checklist
-#. Refactor IStatusQueryable and make monitor service interfaces private
-#. Automatically unmatch remote participants on participant deletion
-#. Handle errors when setting socket buffer sizes
+#. Create InitialConnection for TCP initial peers
+#. Only apply content filter to ALIVE changes
+#. Update version support documentation
+#. Add XML configuration for FlowControllerDescriptor
 #. Github CI management:
 
-    #. Refactor Github CI sanitizer related jobs
-    #. Build Fast DDS Python bindings in Fast DDS Docs Github CI job
-    #. Build ShapesDemo on Ubuntu Github CI
-    #. Fix Python Installation version in Github CI. Address failing system tests environment issues.
-    #. Fix sanitizers CI test summary report
-    #. Run selected VS tool on Windows CI
-    #. Increase sleep to miss the deadline in macOS flaky tests
-    #. Fix ShmTransport buffer recovery MacOS flaky test
-    #. Set fallback branch for ``get_related_branch_from_repo`` correctly
-    #. Add DNS entries to hosts files on github workflows
+    #. Refactor Fast DDS Ubuntu CI to include several tests
+    #. Use token for CCache action
+    #. CI - Avoid CCache in workflows and nightly
+    #. Update README.md with GitHub actions Ubuntu CI nightly
+    #. Build profiling tests as alternate build in Ubuntu CI
```

Comment:
I think we can remove the details about CI improvements, since they are internal 
```suggestion
```